WIP Ab homepage redo

el-mapache opened the pull request:
Fixes issue https://github.com/18F/staffing/issues/638

Proposed changes in this pull request:
- Reworking the homepage design / layout
- Add additional html partials


/cc @amirbey 

I still have a few open questions:

* What urls are expected for each of the links on this page? Some are new links that weren't in the old site

* Moving the copy `Federalist is a product of the General Services Administration.`  to the footer requires a PR into uswds-jekyll. Not sure if that is something we want to do or if there is an alternative way to get the copy in there (without including every footer related file in this repo)

* There are some minor differences in spacing, since the sketch file didnt specify a scale for margin/padding. Should we loop in the designer who worked on this?


[![CircleCI](https://circleci.com/gh/18F/federalist.18f.gov/tree/ab-homepage-redo.svg?style=svg)](https://circleci.com/gh/18F/federalist.18f.gov/tree/ab-homepage-redo)

[:sunglasses: PREVIEW](https://federalist-proxy.app.cloud.gov/preview/18f/federalist.18f.gov/ab-homepage-redo/)

amirbey commented:
@el-mapache - yes please reach to @juliaklindpaintner regarding the "scale for margin/padding"

amirbey commented:
@Bryanthales  is working on posting the content that will be the  destination for the links .... if curious, you can checkout the contents here.... https://docs.google.com/presentation/d/1GEdNySEX4dPalDUjImwEadANPkQe8YLkPvRnvvb08rI/edit#slide=id.g5478bdf7d7_0_6

amirbey commented:
@el-mapache - i noticed some changes on the current case-studies page ... perhaps cascading effects? 
<img width="1311" alt="Screen Shot 2019-08-06 at 11 28 34 AM" src="https://user-images.githubusercontent.com/1261794/62554605-621c8f00-b83f-11e9-845c-8cfd50c69d98.png">

el-mapache commented:
@amirbey Good catch...the designs only address the homepage, and I tried to repurpose the existing classes. I'll go through and fix those.

el-mapache commented:
@juliaklindpaintner I'd love if you could give this a once over. I tried to be as faithful to the designs as I could, though some of the spacing is definitely eyeballed.

Other notable differences:

* I removed the white lines in between the numbers in `how it works` on a tablet. I did this because the designs didn't include mobile screens, and my solution for displaying everything correctly means hiding the lines/numbers on smaller screens.

* I took a guess at how the formatting should look on mobile. I think it works, but you probably have further refinements 😄 

* I moved the `Federalist is a product of...` copy from the center of the footer to the right-hand side. I did this because the footer works by inserting content into a series of slots, and it is pretty difficult to add content that doesn't fit into one of them. This seemed like a decent compromise, let me know if you feel strongly about it and we'll see how we can best address the placement.

Bryanthales commented:
Hey @el-mapache, here is the graphic spacing issue that I brought up to Amir. Basically the (2) header on the timeline is misaligned. Just a slight modification needed to shift the (2) to the right. 

![image](https://user-images.githubusercontent.com/42779803/62806300-50d5bb80-baa7-11e9-838c-f1e2a6b8699b.png)

amirbey commented:
@el-mapache - any updates?

el-mapache commented:
@amirbey So sorry! I will fix this this afternoon and push up the change! Thanks for the feedback 😄

el-mapache commented:
@amirbey @Bryanthales I believe the alignment issues have been fixed!

eddietejeda commented:
What's the status of this?

amirbey commented:
waiting on the summary edits

On Tue, Aug 20, 2019 at 3:16 PM Eddie A Tejeda <notifications@github.com>
wrote:

> What's the status of this?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/18F/federalist.18f.gov/pull/328?email_source=notifications&email_token=AAJUBYVVS5ID4OOQQBF4RLTQFQ7KFA5CNFSM4IJOUJOKYY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOD4XL3FA#issuecomment-523156884>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAJUBYXQ37YQB46MRLOMS7DQFQ7KFANCNFSM4IJOUJOA>
> .
>


-- 
Amir Reavis-Bey
Innovation Specialist, 18F
amir.reavis-bey@gsa.gov

Bryanthales commented:
Reviewed and approved. *hit commit instead of review incorrectly*
